# labels.yaml
compare_policy: 0
detection_threshold: 0.7
label_names:
  - green
  - red
  - yellow
  - blue
histograms:
  green: [0.010489510372281075, 0.019813518971204758, 0.005827505607157946, 0.008741258643567562, 0.11771561950445175, 0.748251736164093, 0.0, 0.0, 0.0, 0.0, 0.0, 0.08916083723306656]
  red: [0.5258562564849854, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.47414371371269226, 0.0, 0.0]
  yellow: [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  blue: [0.01725281961262226, 0.002654280047863722, 0.0, 0.0, 0.0, 0.5089582204818726, 0.1214333102107048, 0.012607830576598644, 0.005972130224108696, 0.002654280047863722, 0.0, 0.32846716046333313]




